cocotb/share/lib/vpi/VpiImpl.cpp

```diff
@@ -157,6 +154,27 @@ gpi_objtype_t to_gpi_objtype(int32_t vpitype)
     }
 }
 
+static gpi_objtype_t const_type_to_gpi_objtype(int32_t const_type)
```

Comment:
Is something similar needed for VHPI/FLI?

Reply:
Nope, they already don't use GPI_PARAMETER.